src/RestClient/endpoints/V3/Customers/Customers.ts

```diff
@@ -0,0 +1,70 @@
+import { AxiosInstance } from 'axios';
+
+import { AxiosPromise } from '../../../../types';
+
+import type { Customer } from './types';
+
+const customersPath = '/v3/customers';
+
+/**
+ * Builds a base Customers API path given an optional customer ID
+ *
+ * @param id Optional filter parameters
+ * @returns Returns either '/v3/customers' or '/v3/customers/:customerId
+ */
+
+export const getCustomersPath = (id?: number | string): string => `${customersPath}/${id ?? ''}`;
```

Comment:
In reference to this comment: https://github.com/bigcommerce/bigcommerce-api-node/pull/21/files#r976792665

Once you change all ID's to be `number`, you can refactor this so that the param `id?: number | string` changes to `id?: number`

Reply:
Thanks! Just added those changes to the types! @matthewvolk